src/config/config.ts

```diff
@@ -114,6 +114,7 @@ export type FlashMessageType = 'error' | 'info' | 'warning' | 'success';
 export interface FlashMessage {
     type: FlashMessageType;
     message: string;
+    title: string | null;
```

Comment:
we usually document `nullable` as `title?: string` rather than `title: string | null`